### PR TITLE
Homepage

### PR DESCRIPTION
**Pull request by H-R-G-P**
Remain top 10 regions on homepage and reorganized code from DatabaseVo.

### CODE REVIEW

File: src/Service/DatabaseService.php [hunk 1]
```diff
@@ -0,0 +1,210 @@
...
+     * @return array<int, Region>
+     */
+    public function getTopTenRegions(): array
+    {
+        $regions = $this->getRegionsSortedByDances();
+
+        $top = [];
+        for ($i = 0; $i < 10; $i++){
+            $top[] = $regions[$i];
+        }
```
Comment (b1514) [inline src/Service/DatabaseService.php:185]: There is a function to slice of array

File: src/Service/DatabaseService.php [hunk 2]
```diff
@@ -0,0 +1,210 @@
+<?php
+
+
```
Comment (b1514) [inline src/Service/DatabaseService.php:3]: Declare in the all files

Comment (H-R-G-P) [reply src/Service/DatabaseService.php:3]: This branch was created before I you create this task, then in master already added declare.

Comment (H-R-G-P) [reply src/Service/DatabaseService.php:3]: But it already is in master branch

Comment (H-R-G-P) [reply src/Service/DatabaseService.php:3]: But it already is in master

Comment (H-R-G-P) [reply src/Service/DatabaseService.php:3]: Git rejects pushing new commits, ask merge at first.

**Comment by b1514** [comment]
This branch cannot be rebased due to conflicts
 - - - -
 Need rebase

**Pull request closed without merging**